In Action Update

Katiegilroyy opened the pull request:
New Quotes added to logo page

Katiegilroyy committed:
In Action Update

New Quotes added to logo page

voborgus commented on content/community/action.md:
```diff
@@ -6,32 +6,113 @@ image: "/images/learn/innersourceinaction.png"
+    {{< company name="3M" image="/images/logos/3m.png" link="https://github.com/customer-stories/3m" author_name="Kevin Truckenmiller" author_title="Lead DevOps Engineer in CRSL" >}}
+      We’re moving towards more openness, which ultimately creates a communication culture and a generative culture, rather than one that’s bureaucratic and process-based.
+      {{< /company >}}
+      {{< company name="ADEO" image="/images/logos/adeo.png" link="https://github.com/customer-stories/adeo" author_name="Guilherme Guitee" author_title="InnerSource and Agility Leader" >}}
```
Guilherme Guitte – typo here

voborgus commented on content/community/action.md:
```diff
@@ -6,32 +6,113 @@ image: "/images/learn/innersourceinaction.png"
-        A project isn’t just about the code, there’s a team behind it, including writers, designers and project managers. [InnerSource] gives you the opportunity to collaborate across multiple time zones and business units.
+      A project isn’t just about the code, there’s a team behind it, including writers, designers and project managers. [InnerSource] gives you the opportunity to collaborate across multiple time zones and business units.
+      {{< /company >}}
+      {{< company name="Walmart" image="/images/logos/walmart.png" link="https://resources.github.com/whitepapers/introduction-to-innersource/" author_name="Jeremy King" author_title="Executive Vice President & Chief Technology Officer for Global eCommerce" >}}
```
IMHO, the title is too long and can break the layout. Can we shorten `Chief Technology Officer` -> `CTO`?

voborgus commented on content/community/action.md:
```diff
@@ -76,23 +160,27 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="IAG" image="/images/logos/iag.png" >}}{{< /company >}}
       {{< company name="IBM" image="/images/logos/ibm.png" >}}{{< /company >}}
       {{< company name="Intuit" image="/images/logos/intuit.png" >}}{{< /company >}}
+      {{< company name="Wiley" image="/images/logos/wiley.png" >}}{{< /company >}}
```
Wiley already listed with quote

voborgus commented on content/community/action.md:
```diff
@@ -114,7 +203,7 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Walmart" image="/images/logos/walmart.png" >}}{{< /company >}}
       {{< company name="Wayfair" image="/images/logos/wayfair.png" >}}{{< /company >}}
-      {{< company name="Wiley" image="/images/logos/wiley.png" >}}{{< /company >}}
+      {{< company name="Wipro" image="/images/logos/wipro.png" >}}{{< /company >}}
```
Wipro already listed with quote

voborgus commented on content/community/action.md:
```diff
@@ -114,7 +203,7 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="US Department of Veteran Affairs" image="/images/logos/Seal_of_the_U.S._Department_of_Veterans_Affairs.svg.png" >}}{{< /company >}}
       {{< company name="Walmart" image="/images/logos/walmart.png" >}}{{< /company >}}
```
Walmart already there at the top

voborgus commented on content/community/action.md:
```diff
@@ -104,6 +192,7 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Standard Charted" image="/images/logos/standard.png" >}}{{< /company >}}
       {{< company name="Stripe" image="/images/logos/stripe.png" >}}{{< /company >}}
       {{< company name="Taxfix" image="/images/logos/taxfix.png" >}}{{< /company >}}
+      {{< company name="Tencent" image="/images/logos/tencent.png" >}}{{< /company >}}
```
Tencent already presented with quote

voborgus commented on content/community/action.md:
```diff
@@ -114,7 +203,7 @@ image: "/images/learn/innersourceinaction.png"
-      {{< company name="Wiley" image="/images/logos/wiley.png" >}}{{< /company >}}
+      {{< company name="Wipro" image="/images/logos/wipro.png" >}}{{< /company >}}
       {{< company name="Zalando" image="/images/logos/zalando.png" >}}{{< /company >}}
       {{< company name="Zendesk" image="/images/logos/zendesk.png" >}}{{< /company >}}
```
zendesk appreared with the quote twice

voborgus commented on content/community/action.md:
```diff
@@ -104,6 +192,7 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Standard Charted" image="/images/logos/standard.png" >}}{{< /company >}}
       {{< company name="Stripe" image="/images/logos/stripe.png" >}}{{< /company >}}
```
Stripe already there

voborgus commented on content/community/action.md:
```diff
@@ -76,23 +160,27 @@ image: "/images/learn/innersourceinaction.png"
-      {{< company name="Nubank" image="/images/logos/nubank.png" >}}{{< /company >}}
+      {{< company name="Nubanks" image="/images/logos/nubank.png" >}}{{< /company >}}
       {{< company name="Otto Group" image="/images/logos/ottogroup.png" >}}{{< /company >}}
+      {{< company name="PayPal" image="/images/logos/paypal.png" >}}{{< /company >}}
```
PayPal doubled

voborgus commented on content/community/action.md:
```diff
@@ -76,23 +160,27 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Navex Global" image="/images/logos/navex-global.png" >}}{{< /company >}}
       {{< company name="Nokia" image="/images/logos/nokia.png" >}}{{< /company >}}
-      {{< company name="Nubank" image="/images/logos/nubank.png" >}}{{< /company >}}
+      {{< company name="Nubanks" image="/images/logos/nubank.png" >}}{{< /company >}}
```
Nubanks? It is already with the quote

voborgus commented on content/community/action.md:
```diff
@@ -114,7 +203,7 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="US Department of Veteran Affairs" image="/images/logos/Seal_of_the_U.S._Department_of_Veterans_Affairs.svg.png" >}}{{< /company >}}
       {{< company name="Walmart" image="/images/logos/walmart.png" >}}{{< /company >}}
       {{< company name="Wayfair" image="/images/logos/wayfair.png" >}}{{< /company >}}
```
Listed twice

voborgus commented on content/community/action.md:
```diff
@@ -76,23 +160,27 @@ image: "/images/learn/innersourceinaction.png"
-       {{< company name="NASA" image="/images/logos/NASA.png" >}}{{< /company >}}
+      {{< company name="NASA" image="/images/logos/NASA.png" >}}{{< /company >}}
       {{< company name="Nab" image="/images/logos/nab.jpg" >}}{{< /company >}}
       {{< company name="Nationwide" image="/images/logos/nationwide.png" >}}{{< /company >}}
```
listed twice

voborgus commented on content/community/action.md:
```diff
@@ -6,32 +6,113 @@ image: "/images/learn/innersourceinaction.png"
-        We need to build a community around the InnerSource projects of people who share the same ideals.
+      We need to build a community around the InnerSource projects of people who share the same ideals.
+      {{< /company >}}
+      {{< company name="IMB" image="/images/logos/ibm.png" link="https://resources.github.com/whitepapers/introduction-to-innersource/" author_name="Jeff Jagoda" author_title="Senior Site Engineer" >}}
```
typo in the company name

Katiegilroyy replied:
Changing Typos now, Thank you

voborgus commented on content/community/action.md:
```diff
@@ -76,23 +160,27 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="HERE Global B.V" image="/images/logos/here.png" >}}{{< /company >}}
+      {{< company name="Huawei" image="/images/logos/huawei.png" >}}{{< /company >}}
       {{< company name="IAG" image="/images/logos/iag.png" >}}{{< /company >}}
       {{< company name="IBM" image="/images/logos/ibm.png" >}}{{< /company >}}
```
appeared twice

voborgus commented on content/community/action.md:
```diff
@@ -76,23 +160,27 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Gravity Infosolutions" image="/images/logos/gravity.png" >}}{{< /company >}}
       {{< company name="HBR" image="/images/logos/hbr.png" >}}{{< /company >}}
       {{< company name="HERE Global B.V" image="/images/logos/here.png" >}}{{< /company >}}
+      {{< company name="Huawei" image="/images/logos/huawei.png" >}}{{< /company >}}
```
appearing twice

voborgus commented on content/community/action.md:
```diff
@@ -50,8 +132,10 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Caprillary" image="/images/logos/capillary.png" >}}{{< /company >}}
+      {{< company name="Comcast" image="/images/logos/comcast.png" >}}{{< /company >}}
       {{< company name="CHR" image="/images/logos/chr.png" >}}{{< /company >}}
       {{< company name="Continental" image="/images/logos/continental.png" >}}{{< /company >}}
```
appearing twice

voborgus commented on content/community/action.md:
```diff
@@ -50,8 +132,10 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="BuzzFeed" image="/images/logos/buzzfeed.png" >}}{{< /company >}}
+      {{< company name="Capital One" image="/images/logos/capitalone.png" >}}{{< /company >}}
       {{< company name="CA Technologies" image="/images/logos/CA_Technologies.png" >}}{{< /company >}}
       {{< company name="Caprillary" image="/images/logos/capillary.png" >}}{{< /company >}}
```
typo and appearing twice

voborgus commented on content/community/action.md:
```diff
@@ -50,8 +132,10 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Bloomberg" image="/images/logos/bloomberg.png" >}}{{< /company >}}
```
already listed with quote

Katiegilroyy replied:
All Companies with Quotes have been listed with the Logos, so we can make presentations and keep them together

voborgus replied:
great! let's click on "resolve" near my comments and check that all companies doubled then (I saw the companies only listed once with quote :))

voborgus commented on content/community/action.md:
```diff
@@ -40,6 +121,7 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Age of Peers" image="/images/logos/ageofpeerstwo.png" >}}{{< /company >}}
       {{< company name="Ahold Delhaise" image="/images/logos/ahold.png" >}}{{< /company >}}
       {{< company name="American Airlines" image="/images/logos/americanairlines.png" >}}{{< /company >}}
+       {{< company name="ASOS" image="/images/logos/asos.png" >}}{{< /company >}}
```
already at the top

voborgus commented on content/community/action.md:
```diff
@@ -40,6 +121,7 @@ image: "/images/learn/innersourceinaction.png"
       {{< company name="Age of Peers" image="/images/logos/ageofpeerstwo.png" >}}{{< /company >}}
       {{< company name="Ahold Delhaise" image="/images/logos/ahold.png" >}}{{< /company >}}
```
appearing twice

voborgus commented on content/community/action.md:
```diff
@@ -6,32 +6,113 @@ image: "/images/learn/innersourceinaction.png"
+      {{< company name="Zendesk" image="/images/logos/zendesk.png" link="https://www.youtube.com/watch?v=vApVU26TZGo" author_name="Jason Smale" author_title="VP of Engineering" >}}
+      Our engineering culture is open and centred around teams owning services and being responsible for running them in production.
       {{< /company >}}
       {{< company name="3M" image="/images/logos/3m.png" >}}{{< /company >}}
```
company doubled

voborgus commented:
Thanks for the update. Sorry for the long list of comments :(

Katiegilroyy commented:
No worries, Thanks for checking!

Katiegilroyy committed:
Update action.md

Typos changed for In Action update, The companies with quotes are also in just logo section to keep them together and for ease when making presentations.

Katiegilroyy committed:
Update action.md

Double Checked there, three on quotes but not on logos.. Thanks

voborgus commented on content/community/action.md:
```diff
@@ -6,32 +6,113 @@ image: "/images/learn/innersourceinaction.png"
+      {{< company name="ASOS" image="/images/logos/asos.png" link="https://medium.com/asos-techblog/adopting-innersource-at-asos-5618435c8d7" author_name="Rob Bell" author_title="Principal Software Engineer" >}}
+      InnerSource has allowed us to deliver dozens of cross-cutting features more efficiently, spanning multiple teams and services. 
+      {{< /company >}}
+      {{< company name="BBC" image="/images/logos/bbc.png" link="https://www.youtube.com/watch?v=pEGMxe6xz-0" author_name="Steph Egan" author_title="Software Engineering Team Lead" >}}
```
Seems there are still companies exists with only 1 occurrence with quote :)

voborgus commented:
Oops, my mistake! Thanks for your patience :))
